editoast/editoast_derive/src/lib.rs

```diff
@@ -237,3 +237,37 @@ pub fn model(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
         .unwrap_or_else(darling::Error::write_errors)
         .into()
 }
+
+#[cfg(test)]
```

Comment:
There is 3 elements exposed below only in case of tests. How about putting them all inside a `mod tests`?

Reply:
Done in 61411e63a. I left the `use` at top-level to be able to access the macro at crate-level: `crate::assert_macro_expansion!(...)` instead of `crate::test_utils::assert_macro_expansion!(...)`.